src/main/java/com/thealgorithms/others/SieveOfEratosthenes.java

```diff
@@ -32,49 +30,39 @@ public class SieveOfEratosthenes {
      * prime numbers till n. Should be more than 1.
      * @return array of all prime numbers between 0 to n
      */
-    public static int[] findPrimesTill(int n) {
-        // Create array where index is number and value is flag - is that number a prime or not.
-        // size of array is n + 1 cause in Java array indexes starts with 0
-        Type[] numbers = new Type[n + 1];
+    public static void findPrimesTill(int n, boolean[] notPrime) {
+        
+        //Initializing indexes 0 & 1 to be true as they are neither prime nor composite.
+        notPrime[0] = true;
+        notPrime[1] = true;
```

Comment:
You have made this branch from your previous branch, not main

Reply:
That's why your other changes are included

Reply:
If the changes seems to be relevant, shall I make a new PR with a fresh branch?